src/node_buffer.cc

```diff
@@ -1208,15 +1208,20 @@ void Initialize(Local<Object> target,
                                    sizeof(*zero_fill_field),
                                    [](void*, size_t, void*){},
                                    nullptr);
-    Local<ArrayBuffer> array_buffer =
+    Local<ArrayBuffer> ab =
         ArrayBuffer::New(env->isolate(), std::move(backing));
     // TODO(thangktran): drop this check when V8 is pumped to 8.0 .
-    if (!array_buffer->IsExternal())
-      array_buffer->Externalize(array_buffer->GetBackingStore());
+    if (!ab->IsExternal())
+      ab->Externalize(ab->GetBackingStore());
+    Maybe<bool> marked = ab
+        ->SetPrivate(env->context(),
+                     env->arraybuffer_untransferable_private_symbol(),
+                     True(env->isolate()));
+    CHECK(!marked.IsNothing());
```

Comment:
You can use `.Check()` on the `Maybe` return value to do the same thing without having to introduce another variable :)

Reply:
Oh i didn't known that. Thank you. Fixed.